common/containers/Tabs/SendTransaction/components/ConfirmationModal.jsx

```diff
@@ -90,6 +123,8 @@ export default class ConfirmationModal extends React.Component {
       }
     ];
 
+    const symbol = token ? token.symbol : 'ETH';
```

Comment:
Can you give me a little more guidance on what this means? Or is this something I can drop in as a TODO for the future when we build out the related feature?